engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/LoggedUser.scala

```diff
@@ -51,4 +51,15 @@ case class CommonUser(id: String,
 case class AdminUser(id: String) extends LoggedUser {
   override def can(category: String, permission: Permission): Boolean = true
   override val isAdmin: Boolean = true
+}
+
+case object ReadOnlyTechUser extends LoggedUser {
```

Comment:
NussknackerInternalUser -> we have one internal user here, let's keep them consistent? This one will be read-only,
but let's put them in one place and with id like NussknackerAdmin, NussknackerReadOnly?